ml-dhs-aws/src/main/java/com/marklogic/client/dhs/JobRunner.java

```diff
@@ -129,10 +216,15 @@ public void run(DatabaseClient client, InputStream csvRecords, InputStream jobCo
          String ingestionStopTime = LocalDateTime.now().toString();
```

Comment:
Should we log the total number of records that were written to the database after the stopJob() call?

Also, should we collect the stop time after writing the after document?